### ISSUE TITLE
Orange/canvas/__init__.py breaks new setuptools as NoneType no longer has a registered loader

### ISSUE THREAD

**Issue by cboulay** [qusp/orange3#1]
I removed python2.7, cleared out its site-packages folder, then installed python3.5.1 with homebrew.
After following the instructions to setup vpe, after attempting `python3 -m Orange.canvas --clear-widget-settings` I'm met with the following error:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Volumes/STORE/Tools/Neurophys/neuropype-community-src-1.2.3/vpe/Orange/canvas/__main__.py", line 388, in <module>
    sys.exit(main())
  File "/Volumes/STORE/Tools/Neurophys/neuropype-community-src-1.2.3/vpe/Orange/canvas/__main__.py", line 221, in main
    if pkg_resources.resource_exists(pkg_name, resource):
  File "/usr/local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1161, in resource_exists
    return get_provider(package_or_requirement).has_resource(resource_name)
  File "/usr/local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1460, in has_resource
    return self._has(self._fn(self.module_path, resource_name))
  File "/usr/local/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1515, in _has
    "Can't perform this operation for unregistered loader type"
NotImplementedError: Can't perform this operation for unregistered loader type
```

After comparing with my Windows install (from latest neuropype executable installer), it appears the main difference is that Windows' `pkg_resources/__init__.py` has a line that explicitly registers `DefaultProvider` as the provider for `NoneType`. This doesn't happen in setuptools anymore after [Jan 26, 2016](https://github.com/pypa/setuptools/commit/397d759e48bc93597c535c2335c9da37178721a5)

I can get vpe to run if I simply comment out the `__loader__ = None` in `vpe/Orange/canvas/__init__.py`. This doesn't appear to affect operation in older setuptools in Windows.

If this breaks vpe for a platform I did not test then it should be possible to make this conditional on the version of setuptools.

**Comment by cboulay** [comment]
Indeed, this line was removed from upstream Orange: 36a814e26e1e7742d1ff20f735610863f9c60306

**Comment by cboulay** [comment]
This was fixed with feb8f94daf2ac01c2157251dd2f520158e755dfd
